### PR TITLE
[TT-3722] Fixes apps cred

### PR DESCRIPTION
**Pull request by cmelgarejo**
- reveal and link-list metas
- section and collection enhancements
  - If a section is empty (no data in the set rows) you can set an Empty message (akin of Note

### CODE REVIEW

File: views/metas/form/action_button.tmpl [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                <input type="hidden" name="{{ .Name }}" value="{{ .Value }}" />
+              {{ end }}
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </form>
+{{ end }}
+</div>
```
Comment (ConsM) [inline views/metas/form/action_button.tmpl:34]: It seems you've mixed up some CSS classes, `modal fade`, `p-{number}` and `justify-content-center` for example are Bootstrap ones, which is not used in the admin app.

File: views/metas/form/reveal.tmpl [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<div class="mdl-textfield mdl-textfield--full-width mdl-js-textfield tyk-form-group--rounded-corners">
```
Comment (ConsM) [inline views/metas/form/reveal.tmpl:1]: are the `mdl-` classes needed ?

**Review by ConsM — CHANGES_REQUESTED** [review]
I've left a couple of comments. 
On a separate note, the script that reveals the value should be refactored but it works for now so as discussed and agreed with Christian we can merge this now and i will refactor it later via [this](https://tyktech.atlassian.net/browse/TT-4108)

**Review by padiazg — APPROVED** [review]
LGTM

**Pull request merged**